### PR TITLE
add module 8 content

### PR DESCRIPTION
**Pull request by tilacog**

### CODE REVIEW

**Review by meaghanlewis — COMMENTED** [review]
@tilacog this last module looks great! Could you just add the knowledge check questions?

File: create-a-commandl-line-program/includes/1-introduction.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Introduction
+
+In this final module, we'll develop a modern, real-world Rust program.
+
+Our application will be a command-line to-do tracker that records our
+tasks into a text file, display them as a list in the terminal and
+lets us mark them complete.
+
```
Comment (meaghanlewis) [inline create-a-commandl-line-program/includes/1-introduction.md:8]: Any helpful learning objectives you can think of?

**Comment by tilacog** [comment]
@meaghanlewis, sorry, I thought we wouldn't need a knowledge check here since it is a "hands-on" module.

Would it be OK to write one with questions about the command-line application project itself?

File: create-a-commandl-line-program/includes/1-introduction.md (hunk 1 above)
Comment (tilacog) [reply create-a-commandl-line-program/includes/1-introduction.md:8]: What about something like:

    Learn how to develop a real-world command-line program using battle-tested third party crates for command line parsing and error handling 

Or

    Learn how to assemble a real-world program using dedicated modules and crates from the Rust community

**Comment by meaghanlewis** [comment]
@tilacog yes absolutely okay to ask a question about the app itself.

File: create-a-commandl-line-program/includes/1-introduction.md (hunk 1 above)
Comment (meaghanlewis) [reply create-a-commandl-line-program/includes/1-introduction.md:8]: I like the first option! I'll add it in now.

**Review by meaghanlewis — APPROVED** [review]

**Pull request merged**